lms/urls.py

```diff
@@ -55,6 +55,10 @@
     admin.site.site_header = _('LMS Administration')
     admin.site.site_title = admin.site.site_header
 
+    if settings.PASSWORD_POLICY_COMPLIANCE_ROLLOUT_CONFIG.get('ENABLE_COMPLIANCE_CHECKING', False):
+        from openedx.core.djangoapps.password_policy.forms import PasswordPolicyAwareAdminAuthForm
+        admin.site.login_form = PasswordPolicyAwareAdminAuthForm
```

Comment:
It may be better to define a custom edx-specific AdminSite somewhere else so that urls.py doesn't need to care about this.